config.json

```diff
@@ -913,6 +913,14 @@
         "practices": [],
         "prerequisites": [],
         "difficulty": 5
+      },
+      {
+        "slug": "binary-search-tree",
+        "name": "Binary Search Tree",
+        "uuid": "39998558-213a-4f92-953e-5764d15ddd7c",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 8
```

Comment:
This might a tad too high I feel. Maybe 6 or 7?

Reply:
I agree 8 was too high, so I lowered it to 6. In the end, the struct definition is not too difficult (e.g. no need for promotion rules, etc.) and the tree traversal is likely medium difficulty both algorithmically and in terms of language sophistication.